docs/source/changes.rst

```diff
@@ -3,18 +3,35 @@ History of changes
 
 .. currentmodule:: h11
 
-v0.8.1 (2018-04-14)
+v0.9.0 (2019-04-22)
 -------------------
 
 Bug fixes:
 
-* Always return headers as ``bytes`` objects (`#60
-  <https://github.com/python-hyper/h11/issues/60>`__)
 * Allow a broader range of characters in header values. This violates
   the RFC, but is apparently required for compatibility with
   real-world code, like Google Analytics cookies (`#57
   <https://github.com/python-hyper/h11/issues/57>`__, `#58
   <https://github.com/python-hyper/h11/issues/58>`__)
+* Constrain regex request-target matching (`#69
+  <https://github.com/python-hyper/h11/pull/69>`__)
+* Handle BrokenStreamError during shutdown grace period (`#71
+  <https://github.com/python-hyper/h11/pull/71>`__)
```

Comment:
This was actually just a docs change, so probably should be split off into a separate section, or at least rephrased to make it clear that it's not a change in h11 itself?

Reply:
Ah yea. I've removed.